code/controllers/subsystem/ticker.dm

```diff
@@ -59,7 +59,10 @@ SUBSYSTEM_DEF(ticker)
 	var/mode_result = "undefined"
 	var/end_state = "undefined"
 
+	//Gamemode setup
+	var/pre_setup_completed = FALSE
 	var/fail_counter
+	var/emergency_start = FALSE
```

Comment:
why not, this is just a failsafe incase extended setup fails (For some reason (Someone breaks it somehow or runtimes a station objective)) and will force the gamemode to bypass roundstart checks. If extended setup is failing then this var tells the gamemode to just ignore the failing setup and go anyway (the same as turning on DEBUG2 mode, but without the other effects since obviously I don't actually want to enable debug2).

Reply:
if someone breaks something it's its own issue.
Extended setup should never fail, if it does we have a problem.